### ISSUE TITLE
Using an LWRP by its name (LogstashForwarder) directly is no longer supported in Chef 13 and will be removed

### ISSUE THREAD

**Issue by runningman84** [parallels-cookbooks/cookbook-logstash-forwarder#7]
can you fix this issue?

```
       [2015-07-02T10:00:59+00:00] WARN: Using an LWRP by its name (LogstashForwarder) directly is no longer supported in Chef 13 and will be removed.  Use Chef::Resource.resource_for_node(node, name) instead.
```

**Comment by Kasen** [comment]
Chef proposes to bypass this warning by using of method: Chef::ResourceResolver.resolve(:mycookbook_myresource). But this method didn't exist in Chef version lower than 12.4.0. I try to bypass this warning and keep the compatibility with older versions ASAP.

**Comment by runningman84** [comment]
what about checking if Chef::ResourceResolver.respond_to?('resolve')

**Comment by kwilczynski** [comment]
@runningman84 which version of the Chef client are you seeing this with?

**Comment by kwilczynski** [comment]
@runningman84 and @Kasen would the suggested Pull Request resolve the issue?

**Comment by Kasen** [comment]
Resolved in #14
